doc/lua_api.md

```diff
@@ -7472,6 +7472,18 @@ child will follow movement and rotation of that bone.
                 abides by, `"custom"` uses `sun_tint` and `moon_tint`, while
                 `"default"` uses the classic Minetest sun and moon tinting.
                 Will use tonemaps, if set to `"default"`. (default: `"default"`)
+        * `fog`: A table with following optional fields:
+            * `fog_distance`: integer, override the client's viewing_range.
+               By default, fog_distance is controlled by the client's viewing_range, and this field is not set.
+               When set the client client can no longer set a larger viewing_range (including range_all).
+               Any value >= 0 sets the desired fog_distance.
+               Any value < 0, resets the behavior to being client-controlled.
+               (default: not set)
```

Comment:
does "not set" mean it returns `nil`?
if I reset it using -1 will it turn into `nil`?

Reply:
Yes on both. Is that too obtuse? I wasn't sure.

On set perhaps is possible to distinguish between not-present and present-but-nil.